verticapy/machine_learning/memmodel/preprocessing.py

```diff
@@ -125,14 +184,74 @@ def __init__(self, mean: ArrayLike, std: ArrayLike) -> None:
 
 class MinMaxScaler(Scaler):
     """
-    InMemoryModel implementation of MinMax scaler.
+    :py:mod:`verticapy.machine_learning.memmodel.base.InMemoryModel` implementation of MinMax scaler.
 
     Parameters
     ----------
-    min_: ArrayLike
+
+    min\_: ArrayLike
         Model's features minimums.
-    max_: ArrayLike
+    max\_: ArrayLike
         Model's features maximums.
+
+    .. note:: :py:mod:`verticapy.machine_learning.memmodel` are defined
+        entirely by their attributes. For example, 'minimum',
+        and 'maximum' values of feature(S) defines a MinMaxScaler model.
```

Comment:
please change the sentence:
... 'maximum' values of the different features define a ...